extract/README.md

```diff
@@ -5,6 +5,7 @@ Various extraction example code.
 - [extract_text_bound.go](extract_text_bound.go) The example showcases how to extract all text for each page along with it's boundary information.
 - [pdf_extract_location.go](pdf_extract_location.go) The example showcases how to extract text at certain location.
 - [pdf_extract_text.go](pdf_extract_text.go) The example showcases how to extract all text for each page of a PDF file.
+- [pdf_simple_extraction.go](pdf_simple_extraction.go) The example showcases how to use a simple extraction procedure in the text extraction process, which skips table processing and topological ordering.
```

Comment:
Would be good to explain why one might want to use that rather than the full more advanced extraction process